src/Controllers/Library/search.php

```diff
@@ -1,6 +1,54 @@
 <?php
-
+/**
+ * Allows URY Librarians to search for Tracks.
+ */
 use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\ServiceAPI\MyRadio_Track;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+
+$form = new MyRadioForm(
+    'lib_search',
+    'Library',
+    'search',
```

Comment:
It might be better to use the module and controller variable for this instead of strings, idk. Some forms do.